data/mods/DinoMod/dinosaur.json

```diff
@@ -544,12 +502,9 @@
     "dodge": 1,
     "armor_bash": 3,
     "armor_cut": 1,
-    "item_chance": 0,
     "luminance": 0,
     "hp": 120,
-    "special_freq": [ 0 ],
-    "death_function": "NORMAL",
-    "special_attack": [ "BOOMER" ],
```

Comment:
maybe this should be "special_attacks": [ "BOOMER", 20 ]
since boomer is a valid special attack, this just doesn't have a valid cooldown.